pkl-formatter/src/test/files/FormatterSnippetTests/output/expr-binary.pkl

```diff
@@ -0,0 +1,19 @@
+foo {
+  123123123123 + 123123123123 * 123123123123 - 123123123123 / 123123123123 +
+    123123123123 ** 123123123123
+  2
+  3
+  4
+}
+
+bar =
+  clazz.superclass == null ||
+    clazz.superclass.reflectee == Module ||
+    clazz.superclass.reflectee == Typed
+
+baz = new Listing {
+  1
+  2
+}
+  |> mixin1
+  |> mixin2
```

Comment:
Hmm. This makes some of my code much more verbose in practice. It'd be nice to allow multiple mixin applications per line (up to the length limit), but use a (future) linter implementation to (ignorably) force one per line.